app/uk/gov/hmrc/economiccrimelevyregistration/models/OtherEntityJourneyData.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.economiccrimelevyregistration.models
+
+import play.api.libs.json.{Json, OFormat}
+
+final case class OtherEntityJourneyData(
+  entityType: Option[OtherEntityType],
+  charityRegistrationNumber: Option[String],
+  companyRegistrationNumber: Option[String],
+  utrType: Option[UtrType],
+  ctUtr: Option[String],
+  saUtr: Option[String],
+  overseasTaxIdentifier: Option[String],
+  postcode: Option[String]
+)
+
+object OtherEntityJourneyData {
+  implicit val format: OFormat[OtherEntityJourneyData] = Json.format[OtherEntityJourneyData]
+
+  def apply(): OtherEntityJourneyData = OtherEntityJourneyData(
```

Comment:
Let's call this `empty()` to align with the FE

Reply:
Removed method as it is not used.